ethcore/src/miner/miner.rs

```diff
@@ -1305,13 +1308,14 @@ impl miner::MinerService for Miner {
 				if self.seal_and_import_block_internally(chain, block) {
 					trace!(target: "miner", "update_sealing: imported internally sealed block");
 				}
+				return
```

Comment:
this return doesn't change anything, right?

Reply:
yeah it doesn't